packages/grid/x-data-grid/src/locales/daDK.ts

```diff
@@ -58,46 +58,46 @@ const daDKGrid: Partial<GridLocaleText> = {
   filterPanelInputPlaceholder: 'Filter værdi',
 
   // Filter operators text
-  filterOperatorContains: 'Indeholder',
-  filterOperatorEquals: 'Lig med',
-  filterOperatorStartsWith: 'Begynder med',
-  filterOperatorEndsWith: 'Ender med',
-  filterOperatorIs: 'Er lig med',
-  filterOperatorNot: 'Er ikke lig med',
-  filterOperatorAfter: 'Efter',
-  filterOperatorOnOrAfter: 'På eller efter',
-  filterOperatorBefore: 'Før',
-  filterOperatorOnOrBefore: 'På eller før',
-  filterOperatorIsEmpty: 'Indeholder ikke data',
-  filterOperatorIsNotEmpty: 'Indeholder data',
+  filterOperatorContains: 'indeholder',
+  filterOperatorEquals: 'lig med',
+  filterOperatorStartsWith: 'begynder med',
+  filterOperatorEndsWith: 'ender med',
+  filterOperatorIs: 'er lig med',
+  filterOperatorNot: 'er ikke lig med',
+  filterOperatorAfter: 'efter',
+  filterOperatorOnOrAfter: 'på eller efter',
+  filterOperatorBefore: 'før',
+  filterOperatorOnOrBefore: 'på eller før',
+  filterOperatorIsEmpty: 'indeholder ikke data',
+  filterOperatorIsNotEmpty: 'indeholder data',
   filterOperatorIsAnyOf: 'indeholder en af',
-  // 'filterOperator=': '=',
-  // 'filterOperator!=': '!=',
-  // 'filterOperator>': '>',
-  // 'filterOperator>=': '>=',
-  // 'filterOperator<': '<',
-  // 'filterOperator<=': '<=',
+  'filterOperator=': '=',
+  'filterOperator!=': '!=',
+  'filterOperator>': '>',
+  'filterOperator>=': '>=',
+  'filterOperator<': '<',
+  'filterOperator<=': '<=',
 
   // Header filter operators text
-  // headerFilterOperatorContains: 'Contains',
-  // headerFilterOperatorEquals: 'Equals',
-  // headerFilterOperatorStartsWith: 'Starts with',
-  // headerFilterOperatorEndsWith: 'Ends with',
-  // headerFilterOperatorIs: 'Is',
-  // headerFilterOperatorNot: 'Is not',
-  // headerFilterOperatorAfter: 'Is after',
-  // headerFilterOperatorOnOrAfter: 'Is on or after',
-  // headerFilterOperatorBefore: 'Is before',
-  // headerFilterOperatorOnOrBefore: 'Is on or before',
-  // headerFilterOperatorIsEmpty: 'Is empty',
-  // headerFilterOperatorIsNotEmpty: 'Is not empty',
-  // headerFilterOperatorIsAnyOf: 'Is any of',
-  // 'headerFilterOperator=': 'Equals',
-  // 'headerFilterOperator!=': 'Not equals',
-  // 'headerFilterOperator>': 'Greater than',
-  // 'headerFilterOperator>=': 'Greater than or equal to',
-  // 'headerFilterOperator<': 'Less than',
-  // 'headerFilterOperator<=': 'Less than or equal to',
+  headerFilterOperatorContains: 'Contains',
```

Comment:
Should this also be `Indeholder`? 🤔 
```suggestion
  headerFilterOperatorContains: 'Indeholder',
```